anti.md

```diff
@@ -58,12 +98,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+You'd have to spend the whole time on the language part because there is no
+systems aspect except following the instructinos on your list.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is pointless and small.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+It isn't a good idea.
```

Comment:
I agree that it's a bit on the small side to be a useful final project for this class. However, I feel like in a world where there are robotic laundry-doers, you could imagine that having a language for communicating laundry instructions would actually be useful.